py_zipkin/logging_helper.py

```diff
@@ -48,14 +43,14 @@ def __init__(
         firehose_handler=None,
     ):
         self.zipkin_attrs = zipkin_attrs
-        self.thrift_endpoint = thrift_endpoint
+        self.endpoint = endpoint
         self.log_handler = log_handler
         self.span_name = span_name
         self.transport_handler = transport_handler
         self.response_status_code = 0
         self.report_root_timestamp = report_root_timestamp
         self.binary_annotations_dict = binary_annotations or {}
-        self.sa_binary_annotations = []
+        self.sa_endpoints = []
```

Comment:
do you mean to be plural?

Reply:
That's something I wasn't sure about. The old code allowed you to set multiple sa binary annotations, so I kept the same behavior for now.

However I don't see why it'd be useful to be able to set more than one "sa". Also I don't know what to do with it in v2, since I can have only 1 remoteEndpoint there...